ttnn/ttnn/operations/conv2d.py

```diff
@@ -33,6 +33,78 @@ def get_conv_output_dim(input, window, stride=1, pad=0, dilation=1):
     return (input + (2 * pad) - dilation * (window - 1) - 1) // stride + 1
 
 
+def prepare_conv_weights(
```

Comment:
whats the point of this wrapping? you could probably just pull up funcs into the namespace you desire

Reply:
To require using kwargs like the current conv functions do